doc/source/programs/gdal_vector_filter.rst

```diff
@@ -0,0 +1,150 @@
+      -o, --output <OUTPUT>                                Output vector dataset [required]
+
+    Common Options:
+      -h, --help                                           Display help message and exit
+      --version                                            Display GDAL version and exit
+      --json-usage                                         Display usage as JSON document and exit
+      --drivers                                            Display driver list as JSON document and exit
+      --config <KEY>=<VALUE>                               Configuration option [may be repeated]
+      --progress                                           Display progress bar
+      --strict                                             Strict mode (default: true)
+                                                           Mutually exclusive with --lax
+      --lax                                                Relaxed mode
+                                                           Mutually exclusive with --strict
+
+    Options:
+      -l, --layer, --input-layer <INPUT-LAYER>             Input layer name(s) [may be repeated]
+      -f, --of, --format, --output-format <OUTPUT-FORMAT>  Output format
+      --co, --creation-option <KEY>=<VALUE>                Creation option [may be repeated]
+      --lco, --layer-creation-option <KEY>=<VALUE>         Layer creation option [may be repeated]
+      --overwrite                                          Whether overwriting existing output is allowed
+      --update                                             Whether to open existing dataset in update mode
+      --overwrite-layer                                    Whether overwriting existing layer is allowed
+      --append                                             Whether appending to existing layer is allowed
+      --output-layer <OUTPUT-LAYER>                        Output layer name
+      --bbox <BBOX>                                        Bounding box as xmin,ymin,xmax,ymax
+      --where <WHERE>|@<filename>                          Attribute query in a restricted form of the queries used in the SQL WHERE statement
+      --fields <FIELDS>                                    Selected fields [may be repeated]
+
+    Advanced Options:
+      --if, --input-format <INPUT-FORMAT>                  Input formats [may be repeated]
+      --oo, --open-option <KEY=VALUE>                      Open options [may be repeated]
+
+
+Description
+-----------
+
+:program:`gdal vector filter` can be used to filter a vector dataset from
+their spatial extent, a SQL WHERE clause or a subset of fields.
+
+``filter`` can also be used as a step of :ref:`gdal_vector_pipeline_subcommand`.
+
+Common options
+++++++++++++++
+
+.. option:: --strict
+
+    Strict mode (default). Error out when a field specified with :option:`--fields`
+    does not exist in the source layer(s).
+
+.. option:: --lax
```

Comment:
I might consider a name that more explicitly describes the behavior, like `--ignore-missing-fields`

Reply:
done